Remotion/Web/ClientScript/Scripts/SmartPage.ts

```diff
@@ -60,6 +60,16 @@ enum SmartPage_DirtyStates
   ClientSide = 'ClientSide'
 }
 
+class SmartPage_ErrorDeclaration {
```

Comment:
Please fix all the braces to be on the next line just like in C#. The idea here is that we want to match the C# style to make switching between C# and JS/TS easier. Please check all the places in this file.